api/server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const server = express();
+server.use(express.json());
+const gamesRoutes = require("../games/games");
```

Comment:
You could alternatively call your `games.js` file `index.js` since it's already in your `games` directory. That would make this:
```javascript
const gamesRoutes = require("../games");
```
since `require` will use an index file if it's present when requiring directories.